tests/PySys/local_mqtt/local_mqtt_pub_sub_retain/run.py

```diff
@@ -0,0 +1,65 @@
+from pysys.basetest import BaseTest
+
+import time
+
+"""
+Validate local publishing with retain flag and subscribing:
+
+Given a configured system
+When we start tedge pub with sudo to publish a message with retain flag to temperature sensor alaram topic
```

Comment:
Can we maybe spare some unrelated info here? E.g. reduce "When we start tedge pub with sudo to publish" to "When we publish ...". That will make it easier to understand the scope of the test

Reply:
Yes, make sense. Shortened them.